pkg/proxy/winkernel/proxier.go

```diff
@@ -146,6 +146,7 @@ type remoteSubnetInfo struct {
 }
 
 const (
+	SOURCE_VIP_NAME      = "source_vip" // The well-known, documented name for the HNS endpoint used as the Source Vip when using overlay network mode
```

Comment:
Have we finalized on this name? Can we provide a link to where this is documented / well-known?

If not documented, should this name be discussed in SIG-Windows meeting to give everyone a chance to comment if there are any concerns?

Reply:
Its not yet finalized. This is one of the areas of this PR where I am seeking feedback on a reasonable name.

Reply:
@tzifudzi lets add this for next weeks sig-windows meeting